### DISCUSSION TITLE
Is there a way to set a default layout?

### DISCUSSION THREAD

**Discussion by vinkla** [slidevjs/slidev#141] [Q&A] [answered]
Say one want to use the same layout throughout the presentation. Is there a way to set a default layout for all slides?

**Comment by antfu** [comment, marked as answer]
Create `layouts/default.vue` and modify as you want. 

Or to apply default configurations for all the slides if you simply want to use the existing layouts.

```md
---
defaults:
  layout: section
---

# Page 1

---

# Page 2

---

# Page 3
```

**Reply by vinkla** [reply]
Thanks Anthony! Keep up the good work 🙌
